plugins/duo_admin/komand_duo_admin/tasks/monitor_logs/task.py

```diff
@@ -57,10 +64,10 @@ def get_parameters_for_query(self, log_type, now, last_log_timestamp, next_page_
                 self.logger.info(f"Subsequent run for {log_type}")
 
             # If for some reason no logs or event have been picked up,
-            # need to ensure that no more than the previous 72 hours is queried - use cutoff check to ensure this
-            last_72_hours = now - timedelta(hours=72)
-            cutoff_time_secs = self.convert_to_seconds(last_72_hours)
-            cutoff_time_millisecs = self.convert_to_milliseconds(last_72_hours)
+            # need to ensure that no more than the previous 30 days is queried - use cutoff check to ensure this
+            max_cutoff_time = now - timedelta(hours=MAX_CUTOFF_HOURS)
```

Comment:
As discussed on slack, I think we had this 72 hours to ensure that if the customer pauses their integration for more than 3 days we won't resume their integration from that timestamp. This could be kept at the 72 hours, with an updated comment as to why (the comment is a bit misleading in my opinion), but will need to call a similar method as `get_filter_time` so that if we've forced a larger date range we honour it until custom_config removes that larger date?